DPG/Tasks/AOTTrack/perfK0sResolution.cxx

```diff
@@ -79,6 +80,8 @@ struct perfK0sResolution {
       return kFALSE;
     if (ntrack.tpcNSigmaPi() > nSigTPC || ptrack.tpcNSigmaPi() > nSigTPC)
       return kFALSE;
+    if (requireTRD && (!ntrack.hasTRD() || !ptrack.hasTRD()))
```

Comment:
Ciao @alessandrobalbino can I ask you even more? 
I would split in two e.g. requireTRDpos and requireTRDneg to have ful control

Reply:
btw also ask about the negative, that the TRD is not available

Reply:
Hi @njacazio, 

I have modified my PR including your suggestions!

Cheers,
Alessandro